mito-ai/src/Extensions/InlineCompleter/provider.ts

```diff
@@ -218,11 +203,16 @@ export class MitoAIInlineCompleter
       if (error) {
         this._notifyCompletionFailure(error);
         throw new Error(
-          `Inline completion failed: ${error.type}\n${error.traceback}`
+          `Inline completion failed: ${error.error_type}\n${error.traceback}`
         );
       }
 
-      return result.list;
+      return {
+        items: result.items.map(item => ({
+          ...item,
+          insertText: this._cleanCompletion(item.content)
+        }))
+      };
```

Comment:
I'm having this happen pretty often where it looks like the server is sending completions, but I'm not seeing any suggestions in the code cell. And then if I like delete a character and add it back or something like that, the completion will show up next time. 

Any thoughts on what's going on here?

https://github.com/user-attachments/assets/da723e71-f910-4fda-a926-8977ece321fb

Reply:
I'll need to investigate this - this gonna take time unfortunately. But I guess it has something to do with the interaction between Mito completer and the JupyterLab completer handler.

Just for my information, which version of JupyterLab were you using?

Reply:
No worries, I'm on jupyterlab 4.3.3